sites/rentmasseur_com/ad_parser.py

```diff
@@ -0,0 +1,182 @@
+import sys
+from typing import List
+import regex as re
+sys.path.append("../../../ads")
+from sites.base_ad_parser import BaseAdParser
+from utils.basic import get_matches
+
+class AdParser(BaseAdParser):
+
+    def is_ad(self) -> bool:
+        profile_actions = self.soup.select_one("div.contactsIconContainer")
+        return profile_actions is not None
+
+    def primary_phone_number(self) -> str:
+        phone_label = self.soup.select_one("div.alert-content span.txtBold.txtXL.block")
+        if phone_label:
+            return phone_label.string.strip()
+        return None
```

Comment:
```suggestion
        return ""
```
Since Fangyi started doing the typing we've been returning empty string in situations like this.

Reply:
Same for others. If it returns a list we just return an empty list: []